gr-uhd/CMakeLists.txt

```diff
@@ -37,6 +42,7 @@ GR_REGISTER_COMPONENT("gr-uhd" ENABLE_GR_UHD
     ENABLE_GR_ANALOG
 )
 message(STATUS "  UHD Version: ${UHD_VERSION}")
```

Comment:
I know this isn't necessarily part of your changes, but I'd ask you add `if(UHD_FOUND)` around these message lines. If UHD isn't found, then we don't want to be printing these!